test/irmin/test_tree.ml

```diff
@@ -398,24 +398,63 @@ let persist_tree : Store.tree -> Store.tree Lwt.t =
   let* () = Store.set_tree_exn ~info:Store.Info.none store [] tree in
   Store.tree store
 
+let inspect =
+  Alcotest.testable
+    (fun ppf -> function
+      | `Contents -> Fmt.string ppf "contents"
+      | `Node `Hash -> Fmt.string ppf "hash"
+      | `Node `Map -> Fmt.string ppf "map"
+      | `Node `Value -> Fmt.string ppf "value")
+    ( = )
+
+let pp_key = Irmin.Type.pp Store.Key.t
+
 let test_clear _ () =
-  let size = 830829 in
-  let* large_tree =
+  (* 1. Build a tree *)
+  let size = 8308 in
```

Comment:
I shrinked the tree. Was it important?

Reply:
yest it was to detect possible stack overflows :-)